home/rvagg/2021-02-exploration-report-small-filecoin-deals.md

```diff
@@ -132,3 +144,15 @@ Deals:
   - f01247 ~0 FIL (0 FIL per epoch)
 * Successfully made a deal with the same miner as for my unverified deal, [`f022352`](https://filfox.info/en/address/f022352). As of writing, still waiting for PreCommit, ~5 hours later. 0 FIL instead of 0.0001458 FIL.
 * Also had success with [`f058369`](https://filfox.info/en/address/f058369) which is also waiting for PreCommit. Costing 0.00081599375 FIL ($0.03).
+
+### Deal outcomes
+
+Successfully got 3 deals into `StorageDealActive` as of final doc edit. This was for a 9.6 Mb file, that turned into a 15.88 MiB padded deal size to consume 16 MiB in the miners' sectors.
+
+* The prices of the deals are `0.00014706421875`, `0` (FIL+ with same miner) and `0.00081599375` (FIL+ with different miner), total of `0.00096305796875`, or $0.0337 at current FIL price.
+* `lotus client balances` shows   `Escrowed Funds: 0.0009616335625 FIL` and `Locked Funds: 0.0009616335625 FIL` - am assuming these are the same thing at this stage of the deal (they were different during deal processing).
+* Unsure where the `0.00000142440625` difference between total deal amount and locked funds amount is - assuming it's an amount already paid out to miners as of the time of writing.
+* The cost to me for making these 3 deals in total was `0.025837882018354755` FIL, or $0.90 at current FIL price. This was mostly consumed getting data into the StorageMarket actor to make the deals.
+* The cost to the miners for making these deals is unaccounted for (it might be an interesting exercise to dig into this and account for their spend and compare it to their earn because there's going to be quite a difference).
+* Question: If you are mostly making deals, can you just transfer a bulk amount into the StorageMarket actor from your wallet so you pay for the transfer once and use it up across many deals?
```

Comment:
☝️ that's definitely a @dirkmc question

Reply:
You can use `lotus wallet market add`:
```
$ lotus wallet market add --help

NAME:
  lotus wallet market add - Add funds to the Storage Market Actor
USAGE:
  lotus wallet market add [command options] <amount>
OPTIONS:
  --from value, -f value   Specify address to move funds from, otherwise it will use the default wallet address
  --address value, -a value Market address to move funds to (account or miner actor address, defaults to --from address)
  --help, -h         show help (default: false)
```

Reply:
Thanks @dirkmc - so if I use this and transfer a whole lot of FIL into there, will I incur any further charges in making deals or do those all fall on the miner now? I think that covers the `AddBalance` transfers here, but what about the `Send` and `Exec`?

<img width="346" alt="Screenshot 2021-03-01 at 9 34 40 pm" src="https://user-images.githubusercontent.com/495647/109485578-f57a7c80-7ad5-11eb-85d6-a55b9edd74d7.png">

Reply:
AFAIK the only message the client needs to send is AddBalance to put the funds required for the deal in escrow with the Storage Market Actor.
Send is for sending FIL to an address, and Exec I believe is when you create a miner. So those messages may not be related to deals your actor has been making.